client/Elements/Components/TopicConsumer/TopicConsumerDetail.tsx

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+import React from 'react';
+import {
+  DrawerPanelContent,
+  DrawerHead,
+  DrawerPanelBody,
+  DrawerActions,
+  DrawerCloseButton,
+  Flex,
+  FlexItem,
+  TextContent,
+  Text,
+  TextVariants,
+} from '@patternfly/react-core';
+import {
+  Table,
+  TableHeader,
+  TableBody,
+  TableVariant,
+} from '@patternfly/react-table';
+import { mockConsumerGroupDetail } from './ConsumerGroupDetail';
+import { ITopicConsumer } from './SearchTopicsConsumerList.patternfly';
+
+export interface ISearchTopicsConsumerListProps {
+  isExpanded: (value: boolean) => void;
+  consumerGroupDetail: ITopicConsumer[];
+}
+
+export const TopicConsumerDetail: React.FunctionComponent<ISearchTopicsConsumerListProps> = ({
+  isExpanded,
+  consumerGroupDetail,
+}) => {
+  const columns = [
+    'Partition',
+    'Client ID + Customer ID',
+    'Current offset',
+    'Log end offset',
+    'Offset lag',
+    {
+      title: '',
+      dataLabel: 'Action',
+    },
+  ];
+
+  const rows = mockConsumerGroupDetail.map((consumergroup) => [
+    consumergroup.partition,
+    consumergroup.id,
+    consumergroup.currentOffset,
+    consumergroup.logendOffset,
+    consumergroup.offsetLag,
+    { title: <a href='#'>Reset offset</a> },
```

Comment:
This should be a button of type link

Reply:
A drawback of using a button was that the drawer became too large to fit into one view, i.e, you had to scroll to see the extreme right buttons. In order to fit everything in one view, I chose to use this. I'll show you the difference between the two in terms of view, maybe we can find a better way :)